rpgJavaInterpreter-core/src/test/resources/RTCALLEEERR.rpgle

```diff
@@ -0,0 +1,9 @@
+     D C               S              1
```

Comment:
Try to add the active group name as below:

```rpgle
     H ACTGRP('MYACT')
```

The test fails, add another test and fix it.